### ISSUE TITLE
Error when using a `node` path with spaces in Windows

### ISSUE THREAD

**Issue by eduherminio** [nfischer/shelljs-plugin-sleep#12]
### Node version (or tell us if you're using electron or some other framework):
16.13.2

### ShellJS version (the most recent version/GitHub branch you see the bug on):
0.8.5

### Operating system:
Windows

### Description of the bug:
node path with spaces (`C:\Program Files\nodejs\node.exe`) doesn't seem to be supported in Windows

```bash
C:\path_to_my_project\api> node ./start.cjs
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
C:\path_to_my_project\node_modules\shelljs\src\common.js:399
        throw e;
        ^

Error [ShellJSInternalError]: Command failed: C:\Program Files\nodejs\node.exe C:\path_to_my_project\node_modules\shelljs-plugin-sleep\sleepHelper.js 10
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.

    at checkExecSyncError (node:child_process:826:11)
    at Object.execSync (node:child_process:900:15)
    at execSleep (C:\path_to_my_project\node_modules\shelljs-plugin-sleep\index.js:12:11)
    at Object.sleepImpl (C:\path_to_my_project\node_modules\shelljs-plugin-sleep\index.js:34:3)
    at Object.sleep (C:\path_to_my_project\node_modules\shelljs\src\common.js:384:25)
    at Object.<anonymous> (C:\path_to_my_project\api\start.cjs:7:7)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
```
### Example ShellJS command to reproduce the error:

```javascript
shell.sleep(1);
```

**Comment by nfischer** [comment]
Oh interesting! What does `shell.exec('echo hello')` look like on your machine? I'm wondering if this bug is unique to the `sleep` plugin or if it's a bug in shell.exec().

**Comment by eduherminio** [comment]
`shell.exec('echo hello')` works as expected @nfischer, as well as `shell.exec("yarn <...>");` commands.

**Comment by nfischer** [comment]
Ah, I think I see the problem. The issue is this line:

https://github.com/nfischer/shelljs-plugin-sleep/blob/4de3c22f2f71ef7726c186c06707ce0cf133f167/index.js#L7

A quick fix will be to swap `shell.config.execPath` for `JSON.stringify(shell.config.execPath)` to ensure that it's wrapped in quotes. An alternative solution might be to migrate to [`child_process.execFileSync()`](https://nodejs.org/api/child_process.html#child_processexecfilesyncfile-args-options).

**Comment by nfischer** [comment]
This should be fixed in v0.2.2 ([changelog](https://github.com/nfischer/shelljs-plugin-sleep/blob/master/CHANGELOG.md#v022-2022-06-19)). Can you try it out and confirm that works for you?

**Comment by eduherminio** [comment]
I just verified that this works as intended now, thanks @nfischer!
